dynamo-lib/src/cmr/dynamo/connection.clj

```diff
@@ -0,0 +1,34 @@
+(ns cmr.dynamo.connection
+  "Contains functions for interacting with the DynamoDB storage instance"
+  (:require [cmr.common.config :as cfg :refer [defconfig]]
+            [taoensso.faraday :as far]))
+
+(def connection-options 
```

Comment:
I am not quite understanding your comment. Is the endpoint adjustable without having to change the software locally?

Reply:
I put the endpoint itself as a defconfig in the config.clj so that it can be adjusted by using an environment variable. It would still need a restart/redeployment to pick up the change though